lightning/src/ln/channel.rs

```diff
@@ -719,13 +730,12 @@ impl<Signer: Sign> Channel<Signer> {
 
 		let background_feerate = fee_estimator.get_est_sat_per_1000_weight(ConfirmationTarget::Background);
 
-		let holder_dust_limit_satoshis = Channel::<Signer>::derive_holder_dust_limit_satoshis(background_feerate);
 		let holder_selected_channel_reserve_satoshis = Channel::<Signer>::get_holder_selected_channel_reserve_satoshis(msg.funding_satoshis);
-		if holder_selected_channel_reserve_satoshis < holder_dust_limit_satoshis {
-			return Err(ChannelError::Close(format!("Suitable channel reserve not found. remote_channel_reserve was ({}). dust_limit_satoshis is ({}).", holder_selected_channel_reserve_satoshis, holder_dust_limit_satoshis)));
+		if holder_selected_channel_reserve_satoshis < MIN_DUST_LIMIT_SATOSHIS {
```

Comment:
Since we select the `holder_selected_channel_reserve_satoshis`, could we just ensure that we never select a reserve below `MIN_DUST_LIMIT_SATOSHIS`?

Reply:
Do you mean instead of returning an API error, rouding up the `channel_reserve_satoshis` with MIN_DUST_LIMIT_SATOSHIS.

I think I prefer the user to swallow the error and having manually to bounce up the channel value instead of us doing it automatically. We might silently encroach on its expected liquidity ready to use and falsify higher application logic like an accounting app...No a strong opinion here.

Reply:
It just seem like it doesn't make sense for `get_holder_selected_channel_reserve_satoshis` to ever return a value less than `330`. But, fine to leave that for follow-up

Reply:
I don't think our API prevent a user to try a `new_outbound` with less than 330 sat ? And if does so `get_holder_selected_channel_reserve_satoshis` will return the exact value. 

That said there is a TODO to make more sense of `get_holder_selected_channel_reserve_satoshis`. We can address it at that time.